src/lib/viewers/media/DashControls.tsx

```diff
@@ -63,14 +81,30 @@ export default function DashControls({
                 onTimeChange={onTimeChange}
             />
 
-            <div className="bp-DashControls-bar">
-                <div className="bp-DashControls-group">
+            <div
+                className={classNames('bp-DashControls-bar', {
+                    'bp-DashControls-bar--two-groups': !videoAnnotationsEnabled,
+                    'bp-DashControls-bar--three-groups': videoAnnotationsEnabled,
+                })}
+            >
+                <div className="bp-DashControls-group bp-AnnotationsControls-group">
+                    {videoAnnotationsEnabled && (
+                        <AnnotationsControls
+                            annotationColor={annotationColor}
+                            annotationMode={annotationMode}
+                            hasDrawing
+                            hasRegion
+                            onAnnotationModeClick={onAnnotationModeClick}
+                            onAnnotationModeEscape={onAnnotationModeEscape}
+                        />
+                    )}
+                </div>
+                <div className="bp-DashControls-group bp-DashControls-group--play-pause">
```

Comment:
Should we be following the SUIT convention? So something like `bp-DashControls-play-pause` Do we need multiple classnames like this? And If we do need multiple classnames, we can possibly use `classNames('bp-DashControls-group', 'bp-DashControls-group--play-pause')` for consistency